### PR TITLE
Add methods for schedule slots in Model interface

### PR DESCRIPTION
**Pull request by limdylan**

### CODE REVIEW

**Comment by limdylan** [comment]
Only definitions are present so far; methods are not implemented yet.

File: src/main/java/seedu/address/model/slot/Slot.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    public LocalDateTime getDateTime() {
+        return dateTime;
+    }
+
+    public Duration getDuration() {
+        return duration;
+    }
+
+    public LocalDateTime getEndDateTime() {
+        //TODO
```
Comment (umaikaze) [inline src/main/java/seedu/address/model/slot/Slot.java:47]: I know this is a dummy but later can just dateTime.plus(duration), it works for my ip at least

**Review by umaikaze — APPROVED** [review]
Very nice skeleton can merge asap alr

File: src/main/java/seedu/address/model/slot/Slot.java (hunk 1 above)
Comment (limdylan) [reply src/main/java/seedu/address/model/slot/Slot.java:47]: yep!

**Review by umaikaze — APPROVED** [review]
Very insightful changes :)

**Pull request merged**